BambooTracker.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=BambooTracker
+Comment=Tracker for YM2608 (OPNA) which was used in NEC PC-8801/9801 series computers
```

i18n can't hurt, right? ;)
```suggestion
Comment=Tracker for YM2608 (OPNA) which was used in NEC PC-8801/9801 series computers
Comment=A tracker for the YM2608 (OPNA), which was used in NEC PC-8801/9801 series of computers.
Comment[jp]=このアプリケーションはNEC PC-8801/9801シリーズに搭載されていたFM音源YM2608(OPNA)向けのトラッカーです。
Comment[de]=Ein Tracker für den YM2608 (OPNA), welcher in NECs PC-8801/9801 Serien an Computern verwendet wurde.
```